index.html

```diff
@@ -5,43 +5,43 @@
     <link href="assets/css/style.css" rel="stylesheet">
 </head>
 <body>
-    <div class="header">
-        <div class="navigation-panel">
+    <header class="header">                                        
+        <nav class="header__navigation-panel">
             <a href="" class="logo">
                 <img src="assets/css/img/Vector.png"></img>
             </a>
-            <nav>
-                <ul class="navigation-list">
-                    <li>
-                        <a href="">features</a>
-                    </li>
-                    <li>
-                        <a href="">premium</a>
-                    </li>
-                    <li>
-                        <a href="">pricing</a>
-                    </li>
-                    <li>
-                        <a href="">about</a>
-                    </li>
-                    <li>
-                        <a href="">sign in</a>
-                    </li>
-                    <li>
-                        <a href="">get started</a>
-                    </li>
-                </ul>
-            </nav>
-        </div>
-        <div class="center-text"> 
-            <div class="text">
+            <ul class="header__navigation-list">
```

Comment:
Only `ul` should be wrapped in `nav`. `nav` should use mix it should be a block and an element of header